src/app.ts

```diff
@@ -195,19 +205,21 @@ async function runWizard() {
     }
   }
 
-  // Ask for Anthropic key
+  // Ask for AI API key
 
-  wizardState.anthropicKey =
+  wizardState.smarterApikey =
     (await password({
       message:
-        "Please enter an Anthropic API key.\n (You can leave this blank if it's already in the ENV variable.): ",
+        "Please enter your API key.\n (You can leave this blank if it's already in the ENV variable.): ",
```

Comment:
We should probably ask with the specific provider name - less confusing to users.